marklogic-data-hub/src/main/java/com/marklogic/hub/impl/DataHubImpl.java

```diff
@@ -246,53 +246,42 @@ public boolean isServerVersionValid(String versionString) {
                 versionString = versions.getMarkLogicVersion();
             }
             int major = Integer.parseInt(versionString.replaceAll("([^.]+)\\..*", "$1"));
-            if (major < 9) {
+            if (major != 9) {
                 return false;
             }
             boolean isNightly = versionString.matches("[^-]+-(\\d{4})(\\d{2})(\\d{2})");
             //Support any 9.0 version > 9.0-5 and all 10.0 versions.
-            if (major == 9) {
```

Comment:
We probably have to update the comment as well?

Reply:
Updated the comment